.coveragerc

```diff
@@ -142,6 +142,9 @@ omit =
     homeassistant/components/maxcube.py
     homeassistant/components/*/maxcube.py
 
+    homeassistant/components/melissa.py
```

Comment:
Since you added tests (Thank you!) no need to add those lines.

Reply:
Was unsure if you just wanted to measure core ;)

Reply:
We want to measure everything testable :)

Reply:
And also, tests are great to test the sanity of ones mind :D